opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/ReportTypeListRegisterActivity.java

```diff
@@ -0,0 +1,68 @@
+package org.smartregister.giz.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.smartregister.domain.FetchStatus;
+import org.smartregister.giz.R;
+import org.smartregister.giz.model.ReportGroupingModel;
+
+import java.util.ArrayList;
+
+public class ReportTypeListRegisterActivity extends ReportRegisterActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        ListView listView = findViewById(R.id.lv_reportRegister_groupings);
+        TextView titleTv = findViewById(R.id.title);
+
+        if (titleTv != null) {
+            titleTv.setText(R.string.reports_type);
+        }
```

Comment:
you can move this to a `setUpViews()` method that will then be inherited by this class

Reply:
Done